config/auth_conf.xml.sample

```diff
@@ -88,12 +104,14 @@
             <bind-password>{password}</bind-password>
             <auto-register-username>{sAMAccountName}</auto-register-username>
             <auto-register-email>{mail}</auto-register-email>
+            <auto-register-roles>gidNumber</auto-register-roles>
 -->
             <!-- For OpenLDAP: -->
 <!--        <bind-user>{dn}</bind-user>
             <bind-password>{password}</bind-password>
             <auto-register-username>{uid}</auto-register-username>
             <auto-register-email>{mail}</auto-register-email>
+            <auto-register-roles>gid</auto-register-roles>
```

Comment:
It's not readily apparent what this value does - the LDAP response's `gid` value is used as the name for the Galaxy group and/or role? Why is this not templated like other values?